net/dnscache/dnscache.go

```diff
@@ -205,34 +201,34 @@ func (r *Resolver) LookupIP(ctx context.Context, host string) (ip, v6 net.IP, al
 			if debug {
 				log.Printf("dnscache: error resolving %q: %v", host, res.Err)
 			}
-			return nil, nil, nil, res.Err
+			return netip.Addr{}, netip.Addr{}, nil, res.Err
 		}
 		r := res.Val
 		return r.ip, r.ip6, r.allIPs, nil
 	case <-ctx.Done():
 		if debug {
 			log.Printf("dnscache: context done while resolving %q: %v", host, ctx.Err())
 		}
-		return nil, nil, nil, ctx.Err()
+		return netip.Addr{}, netip.Addr{}, nil, ctx.Err()
```

Comment:
maybe create 
`var zero netip.Addr`
and return that instead? it'll make it easier to read

Reply:
Or `zip` :)

Reply:
`zaddr`? :D

Reply:
It will always be `netaddr.IP` to me :)